pkg/virt-api/webhooks/validating-webhook/admitters/vmi-create-admitter.go

```diff
@@ -220,8 +220,12 @@ func validateNetworksMatchInterfaces(field *k8sfield.Path, spec *v1.VirtualMachi
 	portForwardMap := make(map[string]struct{})
 
 	vifMQ = spec.Domain.Devices.NetworkInterfaceMultiQueue
-	isVirtioNicRequested = false
 
+	if len(spec.Networks) == 0 && len(spec.Domain.Devices.Interfaces) == 0 {
+		return networkInterfaceMap, vifMQ, true, causes, done
+	}
+
+	isVirtioNicRequested = false
```

Comment:
This is redundant.

Reply:
because it is named variable ?
It was there also before btw, it is explicit

Reply:
Just suggestion.